src/block/block_open.c

```diff
@@ -82,33 +82,31 @@ __wt_block_manager_create(WT_SESSION_IMPL *session, const char *filename, uint32
 }
 
 /*
- * __block_destroy --
+ * __wt_block_close --
  *     Destroy a block handle.
```

Comment:
Should this now say "Close a block handle"?

Reply:
Fixed in 8f571b2e9.